format/format_test.go

```diff
@@ -0,0 +1,178 @@
+package format_test
+
+import (
+	"testing"
+
+	"github.com/sketch-hq/gql-lint/format"
+	"github.com/sketch-hq/gql-lint/output"
+)
+
+var input output.Data = output.Data{
```

Comment:
that's a bit weird that we're saving `output.Data` into `input`, right? Giving it some though, the only sensible change I can come up with is to rename `output` to `result`, but it's not killing it for me either. Up to you :)

Reply:
haha, that was also something I struggled with. But the input of the formater is the output data from the tool. But I can use `data` instead `input` and it will look less painful

Reply:
ah good one 👌